pkg/cfg/config.go

```diff
@@ -231,6 +234,23 @@ func (c *config) GetStringSlice(key string) []string {
 	return strSlice
 }
 
+func (c *config) GetTime(key string) time.Time {
+	c.lck.Lock()
+	defer c.lck.Unlock()
+
+	c.keyCheck(key)
+
+	data := c.get(key)
+	tm, err := cast.ToTimeE(data)
+
+	if err != nil {
+		c.err(err, "can not cast value %v[%T] of key %s to []string", data, data, key)
```

Comment:
I find this error message misleading. You're not actually casting anything to a string slice. And the type looks like a slice/map access operation.
Instead, I suggest going with something like `can not cast value %v [type %T] of key %s to time.Time`

Reply:
fixed